index.html

```diff
@@ -12736,11 +12736,12 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				<p><a>Defines</a> a human-readable, author-localized description for the <a>role</a> of an <a>element</a>.</p>
 				<p>Some <a>assistive technologies</a>, such as screen readers, present the role of an element as part of the user experience. Such assistive technologies typically localize the name of the role, and they may customize it as well. Users of these assistive technologies depend on the presentation of the role name, such as "region," "button," or "slider," for an understanding of the purpose of the element and, if it is a widget, how to interact with it.</p>
 				<p>The <code>aria-roledescription</code> property gives authors the ability to override how assistive technologies localize and express the name of a role. Thus inappropriately using <code>aria-roledescription</code> may inhibit users' ability to understand or interact with an element. Authors SHOULD limit use of <code>aria-roledescription</code> to clarifying the purpose of non-interactive container roles like <rref>group</rref> or <rref>region</rref>, or to providing a <em>more specific</em> description of a <rref>widget</rref>.</p>
-				<p> When using <code>aria-roledescription</code>, authors SHOULD also ensure that:</p>
+				<p>When using <code>aria-roledescription</code>, authors SHOULD also ensure that:</p>
 				<ol>
 					<li>The element to which <code>aria-roledescription</code> is applied has a valid <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> role or has an implicit WAI-ARIA role semantic.</li>
 					<li>The value of <code>aria-roledescription</code> is not empty or does not contain only whitespace characters.</li>
 				</ol>
+				<p class="note">Depending on the assistive technology, user verbosity settings, or other factors, certain elements and their implicit or explicit ARIA role descriptions might not be conveyed. Authors need to be aware that if specifying <code>aria-roledescription</code> on these elements, then their custom role descriptions may not be conveyed by these assistive technologies, as well.</p>
```

Comment:
reads awkwardly - would it be better without the last "as well"?

Reply:
agreed.  definitely needed to step away from that for a bit.  have tried to make it less bad.